spec/javascripts/miq_grid_spec.js

```diff
@@ -1,54 +0,0 @@
-describe('miq_grid.js', function () {
```

Comment:
I guess you didn't want to remove the original file

Reply:
Added back. Thanks 👍